exercises/split-second-stopwatch/canonical-data.json

```diff
@@ -3,7 +3,14 @@
   "comments": [
     "All times are formatted in '<hours>:<minutes>:<seconds>' format.",
     "Hours, minutes, and seconds are formatted using two digits.",
-    "Thus 4 hours, 23 minutes, and 6 seconds is represented as '04:23:06'."
+    "Thus 4 hours, 23 minutes, and 6 seconds is represented as '04:23:06'.",
+    "Tracks are free to convert these times into a format that is the most ",
```

Comment:
Some of these lines have trailing whitespace while others do not. I assume that can be made consistent.

Reply:
Fixed